index.html

```diff
@@ -198,6 +198,24 @@ <h3>Expression Format</h3>
           [=preference=] for the Global Privacy Control value.
         </p>
       </section>
+      <section>
+        <h3>Preference Caching</h3>
+        <p>
+          The [=preference=] MUST be cached on every top-level navigation to ensure consistency in communication of
+          the person's request that their data "not be sold or shared." This means that if the [=preference=] changes
+          during or after a top-level navigation, it WILL NOT be reflected until the next navigation.
+        </p>
+        <p>
+          A [=top-level browsing context=] has a <dfn><code>cachedGPCAtTimeOfLastNavigation</code></dfn> boolean.
+          It is initially <code>false</code>.
+        </p>
+        <p>
+          When [=process a navigate fetch=] is run with [=top-level browsing context=] as <code>browsingContext</code>,
+          [=top-level browsing context=]'s <dfn><code>cachedGPCAtTimeOfLastNavigation</code></dfn> is updated to
```

Comment:
This is a tiny bit of a tautology.  You can just call it "gpcAtNavigation" or something that is less of a mouthful.